gitlab/datadog_checks/gitlab/metrics.py

```diff
@@ -124,19 +124,33 @@
     "geo_job_artifacts_synced_missing_on_primary": "geo.job_artifacts_synced_missing_on_primary",
     "geo_attachments_synced_missing_on_primary": "geo.attachments_synced_missing_on_primary",
     "geo_repositories_checksummed_count": "geo.repositories_checksummed_count",
+    "geo_repositories_checksummed": "geo.repositories_checksummed",
     "geo_repositories_checksum_failed_count": "geo.repositories_checksum_failed_count",
+    "geo_repositories_checksum_failed": "geo.repositories_checksum_failed_count",
     "geo_wikis_checksummed_count": "geo.wikis_checksummed_count",
+    "geo_wikis_checksummed": "geo.wikis_checksummed_count",
     "geo_wikis_checksum_failed_count": "geo.wikis_checksum_failed_count",
+    "geo_wikis_checksum_failed": "geo.wikis_checksum_failed_count",
     "geo_repositories_verified_count": "geo.repositories_verified_count",
+    "geo_repositories_verified": "geo.repositories_verified_count",
     "geo_repositories_verification_failed_count": "geo.repositories_verification_failed_count",
+    "geo_repositories_verification_failed": "geo.repositories_verification_failed_count",
     "geo_repositories_checksum_mismatch_count": "geo.repositories_checksum_mismatch_count",
+    "geo_repositories_checksum_mismatch": "geo.repositories_checksum_mismatch_count",
     "geo_wikis_verified_count": "geo.wikis_verified_count",
+    "geo_wikis_verified": "geo.wikis_verified_count",
     "geo_wikis_verification_failed_count": "geo.wikis_verification_failed_count",
+    "geo_wikis_verification_failed": "geo.wikis_verification_failed_count",
```

Comment:
So as I understand, all these new metrics in 705694d5237118122e85ad3b1addff1b6250a17e correspond to Gitlab v13 renaming from `<name>_count` to just `<name>`, correct?